sconstruct

```diff
@@ -88,18 +89,23 @@ def validateVersionNumber(key, val, env):
 
 
 vars = Variables()
-vars.Add("version", "The version of this build", buildVars.addon_info["addon_version"])
-vars.Add("versionNumber", "Version number of the form major.minor.patch", "0.0.0", validateVersionNumber)
+vars.Add("version", "The version of this build",
+         buildVars.addon_info["addon_version"])
+vars.Add("versionNumber", "Version number of the form major.minor.patch",
+         "0.0.0", validateVersionNumber)
 vars.Add(BoolVariable("dev", "Whether this is a daily development version", False))
-vars.Add("channel", "Update channel for this build", buildVars.addon_info["addon_updateChannel"])
+vars.Add("channel", "Update channel for this build",
```

Comment:
@josephsl Should we still maintain this field, since now we know it won't be used by the store? I'm sure many of us are displeased with that, but I know some are fine with it. Regardless, NV Access seems to have no desire to start honoring it.

Reply:
these changes just happened with my autoformatter. I can remove them if desired but removing this field should not be part of this PR as it has nothing to do with the task I was aiming for here and a rollback of this change would be cleaner without unrelated changes.